javascript/javascript1/week1/jsweek1.js

```diff
@@ -0,0 +1,68 @@
+// Age-ify (A future age calculator)
+let yearOfBirth = 1999;
+let yearFuture = 2060;
+
+let age = yearFuture - yearOfBirth;
```

Comment:
The value of these variables never changes in the code, so you can actually use `const`s for these :)